packages/gp2-components/src/organisms/ProjectCard.tsx

```diff
@@ -72,6 +78,11 @@ const ProjectCard: React.FC<ProjectCardProps> = ({
     >
       {title}
     </LinkHeadline>
+    {tags.length > 0 && (
+      <div css={tagContainerStyles}>
+        <TagList tags={tags.map((tag) => tag.name)} />
```

Comment:
This is also limited to 3
<img width="627" alt="Screenshot 2024-03-25 at 14 17 21" src="https://github.com/yldio/asap-hub/assets/16595804/53edb06d-ff5f-462a-a2d2-007f30809ba1">